src/variant/IVariantFeatureManager.ts

```diff
@@ -0,0 +1,6 @@
+import { ITargetingContext } from "../common/ITargetingContext";

```

Comment:
I don't know if there is value in keeping `IVariantFeatureManager` and `IFeatureManager` separate. We did it in .net so because adding `getVariant` to `IFeatureManager` could be a breaking change. I imagine we would just have a single interface `IFeatureManager` (or no interface if we have no reason for it)

Reply:
So far, as you mentioned, it's identical (functionality wise) to putting everything into the impl aka `FeatureManager` without these interfaces. 
These two interfaces are internal and basically acting as definition of public APIs for v1 and v2. It's clearer than putting everything in feature manager, and no side effect.

> I imagine we would just have a single interface IFeatureManager (or no interface if we have no reason for it)

Thanks for explaining about the interfaces in .NET. I'm also OK with using one interface. It makes more sense.

Reply:
ca66f77 : simply use a single `IFeatureManager` interface